src/store/getters.js

```diff
@@ -122,9 +122,9 @@ export default {
   ),
   getTxDirection: (_, { account: { address }, getTxType }) => ({ tx }, externalAddress) => {
     if (getTxType({ tx }) === SCHEMA.TX_TYPE.spend) {
-      return tx.recipientId === address
-        ? TX_FUNCTIONS.received
-        : TX_FUNCTIONS.sent;
+      return tx.senderId === address
+        ? TX_FUNCTIONS.sent
+        : TX_FUNCTIONS.received;
```

Comment:
With that change you'll see that transaction as received
https://feature-fix-transaction-displaying.wallet.z52da5wt.xyz/account-details/transactions/th_2JwMpEBBQP5DgE75MhRaTAtXMmJwriRZDpsZUV1qyoGiy2myvg

Reply:
I will omit this change from the PR in preference to more general issue #1809